src/components/tables/table-children/TableChildBody.tsx

```diff
@@ -1,70 +1,90 @@
-    const rowWrapperId = `${getTableChildComponentId(props.id, TableChildComponent.TABLE_ROW_WRAPPER)}${rowData.id}`;
-    const headingAndCollapsibleId = `${getTableChildComponentId(props.id, TableChildComponent.TABLE_HEADING_ROW)}${rowData.id}`;
+export interface IInheritedFromTableProps {
+  getActions?: (rowData?: IData) => IActionOptions[];
+  headingAttributes: ITableHeadingAttribute[];
+  collapsibleFormatter?: (tableRowData: IData) => JSXRenderable;
+}
 
-    const tableHeadingRowContent = props.headingAttributes.map((headingAttribute: ITableHeadingAttribute, xPosition: number) => {
-      const { attributeName, attributeFormatter } = headingAttribute;
-      const tableCoordinate = `${xPosition}${yPosition}`;
-      const headingRowContent: JSXRenderable = attributeFormatter
-        ? attributeFormatter(rowData[attributeName], attributeName)
-        : convertUndefinedAndNullToEmptyString(rowData[attributeName]);
+export interface ITableChildBodyProps extends IInheritedFromTableProps {
+  tableId: string;
+  rowData: IData;
+  yPosition: number;
+  isLoading: boolean;
+  onRowClick?: (actions: IActionOptions[]) => void;
+}
 
-      return <td key={tableCoordinate}>{headingRowContent}</td>;
-    });
+export const TableChildBody = (props: ITableChildBodyProps): JSX.Element => {
+  const rowWrapperId = `${getTableChildComponentId(props.tableId, TableChildComponent.TABLE_ROW_WRAPPER)}${props.rowData.id}`;
+  const headingAndCollapsibleId = `${getTableChildComponentId(props.tableId, TableChildComponent.TABLE_HEADING_ROW)}${props.rowData.id}`;
 
-    const collapsibleRowKey = `${getTableChildComponentId(props.id, TableChildComponent.TABLE_COLLAPSIBLE_ROW)}${rowData.id}`;
-    const collapsibleData = props.collapsibleFormatter && props.collapsibleFormatter(rowData, props);
-    const collapsibleRow = collapsibleData
-      ? (
-        <TableCollapsibleRowConnected
-          id={headingAndCollapsibleId}
-          key={collapsibleRowKey}
-          nbColumns={props.headingAttributes.length + TOGGLE_ARROW_CELL_COUNT}>
-          {collapsibleData}
-        </TableCollapsibleRowConnected>
-      )
-      : null;
+  const tableHeadingRowContent = props.headingAttributes.map((headingAttribute: ITableHeadingAttribute, xPosition: number) => {
+    const { attributeName, attributeFormatter } = headingAttribute;
+    const tableCoordinate = `${xPosition}${props.yPosition}`;
+    const headingRowContent: JSXRenderable = attributeFormatter
+      ? attributeFormatter(props.rowData[attributeName], attributeName)
+      : convertUndefinedAndNullToEmptyString(props.rowData[attributeName]);
 
-    const tableRowWrapperClasses = classNames({ 'table-body-loading': !!props.tableCompositeState.isLoading });
+    return (<td key={tableCoordinate}>
+      <div className='wrapper'>{headingRowContent}</div>
+    </td>);
+  });
+
+  const collapsibleRowKey = `${getTableChildComponentId(props.tableId, TableChildComponent.TABLE_COLLAPSIBLE_ROW)}${props.rowData.id}`;
```

Comment:
`const collapsibleRowKey = getTableChildComponentId(props.tableId, TableChildComponent.TABLE_COLLAPSIBLE_ROW) + props.rowData.id;`

I don't see the value of the literal string here

Reply:
I moved the loop out of the file and into the `Table` file. I do not need this line anymore (I've removed it on my side, but haven't pushed yet).